docs/docsite/rst/conf.py

```diff
@@ -238,15 +238,15 @@
     'current_version': version,
     'latest_version': (
         'devel' if tags.has('all') else
-        '2.16' if tags.has('core_lang') or tags.has('core') else
-        '9' if tags.has('ansible')
+        '2.17' if tags.has('core_lang') or tags.has('core') else
+        '10' if tags.has('ansible')
         else '<UNKNOWN>'
     ),
     # list specifically out of order to make latest work
     'available_versions': (
         ('devel',) if tags.has('all') else
         ('2.15_ja', '2.14_ja', '2.13_ja',) if tags.has('core_lang') else
-        ('2.16', '2.15', '2.14', 'devel',) if tags.has('core') else
+        ('2.17', '2.16', '2.15', 'devel',) if tags.has('core') else
```

Comment:
Can't make this change until release day as well, but it could go in the same PR that changes line 241 (for core 2.17 release day).